react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,37 +1,33 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Fragment } from 'react';
```

Comment:
You'd have to import everything that comes from the same place, all at once.
`import React, { Fragment } from 'react';`